module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,19 @@
+#include "sort.hpp"
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& forwardList) {
+    std::deque<std::string> dq;
+    
+    forwardList.sort( [](const auto str1, const auto str2){
+        if(str1.size() == str2.size()){
+            return str1 < str2;
+        }
+        return str1.size() < str2.size();
+    } );
+    
```

Comment:
In my opinion you could write in one line :
return str1.size() == str2.size() ? str1 < str2: str1.size() < str2.size();